services/search/pkg/search/search_unit_test.go

```diff
@@ -0,0 +1,74 @@
+package search
+
+import (
+	"testing"
+)
+
+func Test_parseScope(t *testing.T) {
```

Comment:
Any reason you chose the testing framework over ginkgo/gomega? The package is already using ginkgo/gomega and in general I think it would be nice to focus on one framework unless there's a reason not to.

Reply:
done